pkg/controller/controller_utils.go

```diff
@@ -81,6 +82,7 @@ const (
 	//      1+floor(log_2(ceil(N/SlowStartInitialBatchSize)))
 	SlowStartInitialBatchSize = 1
 )
+const DeletePriorityPodAnnotationKey = "controller.alpha.kubernetes.io/delete-priority"
```

Comment:
If this value is not used out of this module, how about using lower case?

Reply:
This part comes from the original commit https://github.com/kubernetes/kubernetes/pull/75763/commits
Feel free to make suggestions.